test/resources/config/env1-1.0.0/asset.yaml

```diff
@@ -1,5 +1,5 @@
-name: test-asset-1
-version: 1.0.0
-type: environment
-spec: spec.yaml
-extra_config: environment.yaml
+name: test-asset-1
+version: 1.0.0
+type: environment
+spec: spec.yaml
+extra_config: environment.yaml
```

Comment:
These look like line ending changes. Would you please revert these and related changes to the test directory in your PR so there's no confusion on what was actually changed?

Reply:
Should be reverted now.